cmd/main.go

```diff
@@ -138,6 +138,7 @@ Formats:
     dots-v2                  experimental dots format, one package per line
     pkgname                  print a line for each package
     pkgname-and-test-fails   print a line for each package and failed test output
+    testdox                  TestDox format (print a sentence for each test)
```

Comment:
Slight preference for removing the parenthesis:

```suggestion
    testdox                  TestDox format, print a sentence for each test
```

Reply:
Hi @dnephin! It's very kind of you to look at this, thank you.

Of course you must make whatever changes you think are appropriate to make the `testdox` format integrate smoothly with your program. The tool I've built around it, `gotestdox`, was conceived as a documentation generator (really, a way to make people name their tests more informatively), not as a test runner, per se. This explains some of the choices that might otherwise be puzzling, such as omitting empty packages, the names of fuzz cases, and (until very recently) failure outputs.

I've noticed that more and more people seem to be using it as their go-to test runner, though, which is weird, but okay. It seems to me that `gotestsum` with `testdox` format gives them the best of both worlds: much more sophisticated and configurable test result processing, but with test names printed in a fun way.